### PR TITLE
BAQE-2097 - Update version.kie and version.kogito to 7.59.0.Final and…

### PR DESCRIPTION
**Pull request by MarianMacik**
… 1.11.1.Final

* So far we have 2 versions, `version.kogito` for Spring Boot and `version.kogito.quarkus` for Quarkus. This is just temporary until we have quarkus-maven-plugin project structure merged.
* I have swapped hardcoded versions of archetypes in integration tests for dynamic properties taken from the parent.
* `kogito-springboot-archetype` is now `kogito-spring-boot-archetype`.

### CODE REVIEW

**Comment by MarianMacik** [comment]
@jiripetrlik this should fix the #10. Your PR is failing because there is no Quarkus archetype with the new version of Kogito as it was switched to quarkus-maven-plugin. So we have to wait until @jstastny-cz will merge Quarkus support.

**Review by jstastny-cz — APPROVED** [review]
LGTM.

**Comment by jiripetrlik** [comment]
@MarianMacik 
Hello and thank you for PR and update!

**Pull request merged**
